### PR TITLE
Migrated node-status to typescript

### PR DESCRIPTION
**Pull request by Malak67**

### CODE REVIEW

**Review by VargaElod23 — APPROVED** [review]

File: services/node-status/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "downlevelIteration": true,
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
```
Comment (VargaElod23) [inline services/node-status/tsconfig.json:7]: Would there be a prob if we'd remove the skipLibCheck?

**Review by VargaElod23 — APPROVED** [review]

File: services/node-status/tsconfig.json (hunk 1 above)
Comment (Malak67) [reply services/node-status/tsconfig.json:7]: Not sure. NextJs adds it by default when creating a new typescript project

**Comment by Malak67** [comment]
@rattrap We need to test this to see if it is still working as before.

File: services/node-status/package.json [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@taraxa_project/taraxa-ui": "^0.0.1",
+    "axios": "^0.26.1",
+    "ethers": "^5.6.0",
+    "@types/node": "18.15.11",
+    "@types/react": "18.0.33",
+    "@types/react-dom": "18.0.11",
```
Comment (rattrap) [inline services/node-status/package.json:17]: Should go in `devDependancies`

**Review by rattrap — CHANGES_REQUESTED** [review]

**Review by rattrap — APPROVED** [review]

**Pull request merged**